YSI_Game/y_weapondata/y_weapondata_entry.inc

```diff
@@ -204,7 +209,7 @@ static stock YSI_g_WeaponFireType[47] = {
 
 stock Weapon_GetFireType(weaponid)
 {
-	return Weapon_IsValid(weaponid) ? YSI_g_WeaponFireType[weaponid] : (-1);
+	return YSI_Weapon_IsValid__(weaponid) ? YSI_g_WeaponFireType[weaponid] : (-1);
```

Comment:
Just use `Weapon_IsValid`

Reply:
For some reason `stock Weapon_IsValid` gives an invalid instruction run-time error. It only works if it is a macro.